docker/debian.Dockerfile

```
@@ -0,0 +1,115 @@
+# -------------------------------------
+#  PrestaShop FlashLight: Debian image
+# -------------------------------------
+ARG PS_VERSION
+ARG PHP_VERSION
+ARG PHP_FLAVOUR
+FROM php:${PHP_FLAVOUR} AS base-prestashop
+ARG PS_VERSION
+ENV PS_FOLDER=/var/www/html
+
+# Install base tools
+RUN apt-get update \
+  && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -qqy \
+  bash less vim git tzdata zip unzip curl jq netcat-traditional ca-certificates \
+  lsb-release libgnutls30 gnupg libmcrypt4 libiconv-hook1 \
+  nginx libnginx-mod-http-headers-more-filter libnginx-mod-http-geoip \
+  libnginx-mod-http-geoip libnginx-mod-stream \
+  && apt-get clean
+
+# Install PHP requirements
+# see: https://olvlvl.com/2019-06-install-php-ext-source
+# see: https://stackoverflow.com/a/73834081
+# see: https://packages.sury.org/php/dists/
+RUN curl -sSLo /usr/share/keyrings/deb.sury.org-php.gpg https://packages.sury.org/php/apt.gpg \
+  && . /etc/os-release \
+  && echo "deb [trusted=yes] https://packages.sury.org/php/ ${VERSION_CODENAME} main" > /etc/apt/sources.list.d/php.list \
+  && rm /etc/apt/preferences.d/no-debian-php;
+RUN apt-get update \
+  && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -qqy \
+  php-gd libghc-zlib-dev libjpeg-dev libpng-dev libzip-dev libicu-dev \
+  && apt-get clean \
```

```suggestion
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* \
```
https://docs.docker.com/develop/develop-images/instructions/#apt-get